### ISSUE TITLE
jx version upgrade doesn't respect bin path and forces `~/.jx/bin/jx`

### ISSUE THREAD

**Issue by vgallissot** [jenkins-x/jx#621]
I used `jx version` to update `jx`.
My `jx` command is in `/usr/local/bin/jx`.
`jx version` updates the cli, but uncompresses the new one in  `~/.jx/bin/jx` without respecting the path of the first one.

```
± which jx
/usr/local/bin/jx

± jx version
NAME               VERSION
jx                 1.1.59
jenkins x platform 0.0.624
kubernetes cluster v1.9.6-gke.0
kubectl            v1.9.3
helm client        v2.8.1+g6af75a8
helm server        v2.8.1+g6af75a8
git                git version 2.14.3

A new jx version is available: 1.1.68
? Would you like to upgrade to the new jx version? Yes
Downloading https://github.com/jenkins-x/jx/releases/download/v1.1.68/jx-linux-amd64.tar.gz to /home/vincent/.jx/bin/jx.tgz...
Downloaded /home/vincent/.jx/bin/jx.tgz

± /home/vincent/.jx/bin/jx version
NAME               VERSION
jx                 1.1.68
jenkins x platform 0.0.624
kubernetes cluster v1.9.6-gke.0
kubectl            v1.9.3
helm client        v2.8.1+g6af75a8
helm server        v2.8.1+g6af75a8
git                git version 2.14.3

± jx version
NAME               VERSION
jx                 1.1.59
jenkins x platform 0.0.624
kubernetes cluster v1.9.6-gke.0
kubectl            v1.9.3
helm client        v2.8.1+g6af75a8
helm server        v2.8.1+g6af75a8
git                git version 2.14.3

A new jx version is available: 1.1.68
? Would you like to upgrade to the new jx version? No
```

**Comment by jstrachan** [comment]
aha!!! It should definitely try find it on the `$PATH` and then copy it there for sure. Great catch thanks!

**Comment by i0n** [comment]
This should be fixed from 1.3.189+
